### DISCUSSION TITLE
About the bug of tool image-resizer

### DISCUSSION THREAD

**Discussion by CracKerMe** [he3-app/he3-feedback#304] [Bug Report]
### Version
image-resizer 1.1.3

### Environment
Windows v2.0.0

### Steps to reproduce
PNG图片调整大小之后保存格式只能为jpg

### What is expected?
根据图片文件的mimetype 保存格式

**Comment by lyzhang1999** [comment]
感谢反馈，已经创建 issues：https://github.com/he3-app/he3-feedback/issues/305
